rollup.config.js

```diff
@@ -26,6 +25,8 @@ function getEsbuild(target) {
   return esbuild({
     minify: false,
     target,
+    external,
+    platform: 'neutral',
```

Comment:
How do these change? I mean, why it was working okay before the change (expecially `external` one, not sure what `platform` does.)

Reply:
I think it's a little more explicitly defined but this is already handled by `tsconfig` ? 
platform `neutral` make sure the esm is not specifically targeted towards browser or node